packages/components/src/autocomplete/autocompleter-ui.tsx

```diff
@@ -62,11 +62,9 @@ function ListBox( {
 					disabled={ option.isDisabled }
 					className={ clsx(
 						'components-autocomplete__result',
-						className,
-						{
-							'is-selected': index === selectedIndex,
-						}
+						className
 					) }
+					variant={ index === selectedIndex ? 'primary' : undefined }
```

Comment:
It might be better to keep the `is-selected` class (without styles) for backwards compatibility.

Reply:
Thanks for the feedback! Restored with 202b055